code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -17,6 +17,8 @@
 	var/check_factions_exactly = FALSE
 	/// Minimum status to attack living beings
 	var/stat_attack = CONSCIOUS
+	/// Do we attack corpses
+	var/attack_corpses = FALSE
```

Comment:
i dont think that is necessary if you just change stat_attack to DEAD

Reply:
thanx for the help i yes